publish.sh

```diff
@@ -12,4 +12,14 @@ publish_branch=publish_$v
 git checkout -b $publish_branch
 git push origin $publish_branch
 
-npx lerna publish
+npx lerna publish --no-push
+
+# Run `npm install` to update the package-lock.json files, copying over the git tags to the
+# new commit.
+tags=$(git tag --points-at HEAD)
+message=$(git log -1 HEAD --pretty=%B)
+npx lerna exec "npm install --ignore-scripts --package-lock-only"
+git commit --all --amend --no-edit -m "(With package-lock.json changes) ${message}"
```

Comment:
You should be able to remove the `--no-edit` flag here - it's saying that the commit message shouldn't be edited, so it's overridden by `-m`

Reply:
Good catch, removed the flag.